test/dotnet-new.Tests/GivenThatIWantANewApp.cs

```diff
@@ -118,7 +118,7 @@ string GetFrameworkPackageVersion()
                     var context = reader.Read(stream);
                     var dependency = context
                         .RuntimeLibraries
-                        .Single(library => library.Name == packageName);
+                        .Single(library => string.Equals(library.Name, packageName, StringComparison.OrdinalIgnoreCase));
```

Comment:
Because the sharedfx's .deps.json file is now being written using a new DependencyModel that doesn't have the "toLower" hack in it.  Thus the Microsoft.NETCore.App.deps.json file now has the normal casing in it. Here the test was expecting the `netstandard.library` library name to be all lower-case, which is not a good assumption.  These are case-insensitive.